editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -1240,9 +1233,9 @@ void SpriteFramesEditor::_update_library(bool p_skip_selector) {
 	if (animated_sprite) {
 		String autoplay_name = animated_sprite->call("get_autoplay");
 		if (autoplay_name.is_empty()) {
-			autoplay->set_pressed(false);
+			autoplay->set_pressed_no_signal(false);
 		} else {
-			autoplay->set_pressed(String(edited_anim) == autoplay_name);
+			autoplay->set_pressed_no_signal(String(edited_anim) == autoplay_name);
```

Comment:
Just leaving a note that this doesn't seem to result in any change in behavior, because the connected method `_autoplay_pressed` doesn't run if `updating` is `true`. So this should be fine.

Reply:
And the same applies to `anim_speed` and `anim_loop`.